packages/dev/core/src/Materials/Node/nodeMaterial.ts

```diff
@@ -342,7 +342,7 @@ export class NodeMaterial extends PushMaterial {
 
     /** Gets or sets the active shader language */
     public override get shaderLanguage(): ShaderLanguage {
-        return this._options.shaderLanguage;
+        return this._options?.shaderLanguage || NodeMaterial.DefaultShaderLanguage;

```

Comment:
```suggestion
        return this._options?.shaderLanguage ?? NodeMaterial.DefaultShaderLanguage;
```

Reply:
There is an interesting overhead when using ?? in UMD, and this is why I only use it when really necessary. In this case, I do need to check if the object itself is there and shaderlanguage should never be falsy

Reply:
Ooh, now I'm curious...

Reply:
?? in umd looks like this:

```javascript
// from this:
let name
const n = name ?? "name";

// you get this
let name;
const n = name !== null && name !== void 0 ? name : "name";
```

However, `||`, as it is a native ES5 feature, just stays `||`. so for size-reasons (and perf in rare cases) it is better to use || if possible, until we move to full es2020 compilation.